Command and Control/DofoilNameCoinServerTraffic.txt

```diff
@@ -0,0 +1,11 @@
+// This is a query to retrieve last 60 days network connections to known Dofoil NameConin servers
+// The full article is available here: https://cloudblogs.microsoft.com/microsoftsecure/2018/04/04/hunting-down-dofoil-with-windows-defender-atp/
+
+NetworkCommunicationEvents 
+| where EventTime > ago(60d)

```

Comment:
The portal currently supports only last 30d lookups...
So, you can remove this filter, and remove/update the comment above

Reply:
Fixed.